### ISSUE TITLE
Delete Version of Controlled Doc - LATEST version label not correctly applied after delete

### ISSUE THREAD

**Issue by gsteimer** [tsgrp/HPI#707]
If you have the following controlled document version tree:
- 1.0, Effective, CURRENT
- 1.1, Draft
- 1.2, Draft, LATEST

And then try to use the Delete Version action to delete the 1.2 version, the version is correctly deleted.  However, the LATEST flag is not applied to the 1.1 version.

This is the job of the `tsg_mark_as_latest` repository method, which is invoked by the `AWControlledDocImpl#doDestroy` TBO method.  Interestingly, it looks like the method code completes sucessfully on the JMS:

```
2015-06-18 14:04:37,481 INFO  [STDOUT] Found version: 1.1
2015-06-18 14:04:37,481 INFO  [STDOUT] Found version: 1.0
2015-06-18 14:04:37,481 INFO  [STDOUT] Latest version: 1.1
2015-06-18 14:04:37,481 INFO  [STDOUT] Marking version as LATEST.
2015-06-18 14:04:37,501 INFO  [STDOUT] Calling super#doSave
2015-06-18 14:04:37,551 INFO  [STDOUT] Finished super#doSave
2015-06-18 14:04:37,551 INFO  [STDOUT] checking to see if policy is attached...
2015-06-18 14:04:37,551 INFO  [STDOUT] Completed doSave TBO for Controlled Doc: 090000088002c768
```

In the logs above, version `090000088002c768` is the 1.1 version for my test, so it looks like all of the code is successfully running.

This will be an issue for any client using our controlled docs model and the delete version action since the checkout action will not be enabled on the 1.1 since it doesn't have the LATEST version label.

**Comment by gsteimer** [comment]
Note that the same thing happens when deleting the 1.1 document.  The 1.0 does not get the LATEST label, even though 1.1 deletes successfully.

**Comment by eschwarzbeck** [comment]
Duplicate of https://github.com/tsgrp/HPI/issues/1078 ?

**Comment by eschwarzbeck** [comment]
Closed in #16155
